tests/integration/podman/move_test_scripts_to_bin.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script moves all shell script files (.sh) from the source directory to
+# the /bin directory in the container, removes the .sh extension, and makes the
+# files executable. This prepares the scripts for use in integration tests.
+#
+# The source directory (/scripts) contains utility scripts necessary for
+# integration tests. These scripts are copied into the container during the
+# Podman build process.
+
+SRC_DIR='/scripts'
+DEST_DIR='/bin'
+
+for file in "$SRC_DIR"/*.sh; do
+  base_name=$(basename "$file" .sh)
+  mv "$file" "${DEST_DIR}/${base_name}"
+  chmod +x "${DEST_DIR}/${base_name}"
+done
```

Comment:
tbh, I don't understand why we need a script for this... you could follow my previous comment and just use the copy commands in the container.